components/blitz/src/ome/formats/model/PixelsProcessor.java

```diff
@@ -44,10 +44,8 @@
 
 /**
  * Processes the pixels sets of an IObjectContainerStore and ensures
- * that the physical pixel dimensions are updated if they were specified by
- * the user. If Image containers are present, Image.acquisitionDate is filled
- * out and that the Image name and description match that which was specified
```

Comment:
@simleo : then perhaps let's also remove the commented code and file a card to come back to the issue on devx?